FirebaseInAppMessaging/Sources/DefaultUI/FIRIAMDefaultDisplayImpl.m

```diff
@@ -72,10 +72,12 @@ + (NSBundle *)getViewResourceBundle {
            [NSBundle mainBundle],
            // Dynamically linked.
            [NSBundle bundleForClass:myClass],
+#if FIREBASE_BUILD_ZIP_FILE
            // Embedded static framework (zip distribution).
```

Comment:
Just to make sure I understand what was going wrong, in the SPM and CocoaPods (static) cases (unlike with the zip distribution) you don't "Embed and Sign" the framework in Xcode so `[NSBundle bundleWithURL:[NSBundle.mainBundle.bundleURL URLByAppendingPathComponent:@"Frameworks/FirebaseInAppMessaging.framework"]]` was `nil`. Is that accurate?

Reply:
precisely!

Reply:
This does require that customers embed our zip distribution's XCFrameworks (or else it will crash with the error I'm fixing for static CocoaPods / SPM), but they should be embedding nowadays.

Reply:
FIAM doesn't have a privacy manifest but it does have a the UI resource bundle, and to prevent the release tooling from packaging different bundles differently (e.g. some SDKs should be embedded, some don't have to), it's spelled out in the zip instructions to embed everything.

Reply:
Sounds good, thanks!